### PR TITLE
Adds `access=permissive` routability scenarios, see #3435.

### PR DESCRIPTION
**Pull request by daniel-j-h**
cc @MoKob

### CODE REVIEW

File: features/car/access.feature [hunk 1]
```diff
@@ -220,3 +220,11 @@ Feature: Car - Restricted access
...
             | highway | toll        | bothw |
             | primary | yes         |       |
+
+    Scenario: Car - Access combinations
+        Then routability should be
+            | highway    | access     | bothw |
+            | primary    | permissive | x     |
+            | steps      | permissive | x     |
+            | footway    | permissive | x     |
+            | garbagetag | permissive | x     |
```
Comment (daniel-j-h) [inline features/car/access.feature]: Looked into this - this scenario is passing at the moment.

See #3435 for a discussion if it should pass or if the highway tag should take precedence.

**Comment by MoKob** [comment]
@daniel-j-h is this still an issue? can we simply merge this, or do you think this should produce different results. I can't really tell from the discussion in #3435

**Comment by daniel-j-h** [comment]
The Wiki says one thing the real world data says another. #3435 is not resolved. My opinion is routing cars over steps makes no sense. Feel free to close this PR - we can revive it once we resolved #3435.

**Comment by TheMarex** [comment]
Replaced by https://github.com/Project-OSRM/osrm-backend/pull/3668

**Pull request closed without merging**